acceptance/config/windows-aio-2012-x86_64.cfg

```diff
@@ -0,0 +1,32 @@
+HOSTS:
+  ubuntu1204:
+    roles:
+      - master
+      - database
+      - dashboard
+    platform: ubuntu-12.04-amd64
+    template: ubuntu-1204-x86_64
+    hypervisor: vcloud
+  w2012-puppetx64:
+    arch: x64
+    roles:
+      - agent
+      - default
+    platform: windows-2012-x86_64
+    template: win-2012-x86_64
+    hypervisor: vcloud
+  w2012-puppetx86:
+    arch: x86
+    roles:
+      - agent
+    platform: windows-2012-x86_64
+    template: win-2012-x86_64
+    hypervisor: vcloud
+CONFIG:
+  type: aio
```

Comment:
The hope was that this file would not be necessary, but without a node config that specifies `type: aio`, Beaker injects the wrong paths based on the pe defaults of `C:\Program Files\Puppet Enterprise\`, and then subsequently can't find `puppet`, `facter`, etc.